packages/core/content-type-builder/admin/src/pages/App/index.js

```diff
@@ -21,12 +21,20 @@ import ContentTypeBuilderNav from '../../components/ContentTypeBuilderNav';
 const ListView = lazy(() => import('../ListView'));
 
 const App = () => {
+  const { startSection } = useGuidedTour();
+  const startSectionRef = useRef(startSection);
```

Comment:
Why do you need a ref here ?

Reply:
mmh I think we needed it before because we really needed the startSection to happen only once and we kept losing the reference of the function, which caused issues! but I tested it and it seems that we don't need it anymore, nice catch thanks!